src/drivers/driver_zephyr.c

```diff
@@ -36,6 +36,63 @@ const struct zep_wpa_supp_dev_ops *get_dev_ops(const struct device *dev)
 	return api->wifi_drv_ops;
 }
 
+#ifdef CONFIG_WIFI_NM_HOSTAPD_AP
+void hostapd_event_wrapper(void *ctx, enum wpa_event_type event, union wpa_event_data *data)
+{
+	struct wpa_supplicant_event_msg msg = { 0 };
+
+	msg.hostapd = 1;
+	msg.ctx   = ctx;
+	msg.event = event;
+	if (data)
+	{
```

Comment:
please place starting `{` to previous line

Reply:
Updated.